src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Files.kt

```diff
@@ -53,6 +55,24 @@ class Files : FileSystem
         return fileList
     }
 
+    override fun getChildFolders(sourceAbsolutePath: String): List<String>
```

Comment:
The existing getAllFilesInFolder methods returns full filenames for all levels under sourceAbsolutePath, whereas iwe want to keep folder path and file name separate. It seemed a bit easier to deal with the recursion in the Controller rather than trying to derive everything from path strings, so have added these methods to get immediate child folders and files.